jenkins/ci.suse.de/cloud-ardana8.yaml

```diff
@@ -2,93 +2,96 @@
     name: cloud-ardana8-gating
     version: 8
     label: cloud-trigger
-    build_pool: cloud-ardana-ci-slot
     jobs:
         - 'cloud-ardana{version}-gating'
```

Comment:
In other jiobs lists below you are using {name} for the job name when it patches the name of the project... Any reason not to do so here?

Reply:
I agree cloud-ardana8-gerrit-x86_64 and cloud-ardana8-gating seem to follow the previous structure the other two jobs use:
 jobs:
        - {name}

And don't define version or arch. It looks like both methods would result in he something, but the code should be consistent.  Unless I an missing something.

Reply:
I see your point. I don't think having two or more templates resolvable via `name` is possible, because JJB wouldn't know which one to pick, but I'll give it a try.

Reply:
Just as I thought, if you have two templates using `name` as a parameter, the jenkins job builder is confused because it doesn't know which one it should use:
```jenkins_jobs.errors.JenkinsJobsException: Duplicate entry found in '/home/snica/PycharmProjects/automation/jenkins/ci.suse.de/templates/cloud-ardana-pipeline-template.yaml: '{name}' already defined```

Reply:
I got it, using different template variables solved it: `ardana_job` for openstack-ardana jobs, `ardana_gerrit_job` for gerrit jobs and `ardana_gating_job` for gating jobs.